xoa/resource_xenorchestra_vm.go

```diff
@@ -183,13 +183,16 @@ func resourceVmCreate(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
-	network_ids := []string{}
+	network_maps := []map[string]string{}
 	networks := d.Get("network").(*schema.Set)
 
 	for _, network := range networks.List() {
 		net, _ := network.(map[string]interface{})
 
-		network_ids = append(network_ids, net["network_id"].(string))
+		network_maps = append(network_maps, map[string]string{
+			"network_id": net["network_id"].(string),
+			"macAddress": net["mac_address"].(string),
```

Comment:
Since the rest of these keys are snake case I would prefer if we made this `mac_address`.